templates/vue/src/components/Lightbox.vue

```diff
@@ -91,6 +90,12 @@ export default {
   computed: {
     ...mapState(["h5pSettings"]),
     ...mapGetters(["getNode"]),
+    node() {
+      return this.getNode(this.nodeId)
+    },
+    canSkip() {
+      return this.node.completed || this.node.skippable
```

Comment:
I think this.node.skippable would return false because it would be undefined for existing nodes. We want to make sure that all existing nodes (and new nodes) are skippable unless the user turns it off and makes it non-skippable. This should be changed to !== false